editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -2052,6 +2052,7 @@ bool CanvasItemEditor::_gui_input_move(const Ref<InputEvent> &p_event) {
 	if (drag_type == DRAG_MOVE || drag_type == DRAG_MOVE_X || drag_type == DRAG_MOVE_Y) {
 		// Move the nodes
 		if (m.is_valid() && !drag_selection.is_empty()) {
```

Comment:
```suggestion
```
You didn't replace this line, breaking the code scopes

Reply:
Could you elaborate on this please?

Reply:
You have
```cpp
if (m.is_valid() && !drag_selection.is_empty()) {
if (!tool_changed && m.is_valid() && !drag_selection.is_empty()) {
```
You can't have these both

Reply:
Oh, never mind, now sure how that happened.